### PR TITLE
fix https://github.com/AdguardTeam/AdguardFilters/issues/89190

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/89190

### CODE REVIEW

File: Filters/exceptions.txt [hunk 1]
```diff
@@ -23,7 +25,7 @@
 @@||ac.ebis.ne.jp^|
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/602
 ! Don't block clicks in AWS emails
-@@||email.awscloud.com^
+@@||email.awscloud.com^|
```
Comment (Alex-302) [inline Filters/exceptions.txt:28]: what blocks them?

Comment (Yuki2718) [reply Filters/exceptions.txt:28]: IDK about `email.awscloud.com`, just noticed it's lacking `|`. `analytics.edgekey.net` is in Spyware and EasyPrivacy, already whitelisted for contents filter in https://github.com/AdguardTeam/AdguardFilters/commit/c16daf177f26e00bf7ccd8c01195e0f8684f405e

Comment (Alex-302) [reply Filters/exceptions.txt:28]: email.awscloud.com is not blocked by our DNS

image

Comment (Yuki2718) [reply Filters/exceptions.txt:28]: @ameshkov https://github.com/AdguardTeam/AdGuardSDNSFilter/commit/53ce5d14788a6e99c2d5cab42963b5bba6664017

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**